source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -79,19 +81,19 @@ largest value of the ``_id`` field as follows:
    .. output:: 
       :language:  console
 
-      FoodOrder(id=1, letter=c, food=coffee with milk)
-      FoodOrder(id=2, letter=a, food=donuts and coffee)
-      FoodOrder(id=3, letter=a, food=maple syrup)
-      FoodOrder(id=4, letter=b, food=coffee with sugar)
-      FoodOrder(id=5, letter=a, food=milk and cookies)
-      FoodOrder(id=6, letter=c, food=maple donut)
+      Order(id=6, date=2022-01-23, orderTotal=10.99, description=1 bagel, 1 orange juice, 1 muffin)
+      Order(id=1, date=2022-01-03, orderTotal=17.86, description=1/2 lb cream cheese and 1 dozen bagels)
+      Order(id=3, date=2022-01-11, orderTotal=19.49, description=1 dozen vanilla cupcakes)
+      Order(id=4, date=2022-01-15, orderTotal=43.62, description=2 chicken lunches and a diet coke)
+      Order(id=5, date=2022-01-23, orderTotal=60.31, description=one large vanilla and chocolate cake)
+      Order(id=2, date=2022-01-11, orderTotal=83.87, description=two medium vanilla birthday cakes)
 
-In the preceding code snippets, we specify our sort criteria using the ``Sorts``
+In the preceding code snippets, we specify the sort criteria using the ``Sorts``
 builder class. While it is possible to specify sort criteria using any class
 that implements the ``Bson`` interface, we recommend that you specify sort
 criteria through the ``Sorts`` builder. For more information on the ``Sorts``
-builder class, see our
-:doc:`guide on the Sorts builder </fundamentals/builders/sort/>`.    
+builder class, see the
+:ref:`guide on the Sorts builder <sorts-builders>`.
```

Comment:
Non-blocking suggestion - I think this is more in line with the majority of the other pages where we have links like this:

```suggestion
:ref:`Sorts builder <sorts-builders>` guide.
```